document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImpl.java

```diff
@@ -24,14 +26,42 @@ public DocumentInfo getDocumentInfo() {
         LOG.info("Started getting document");
 
         String resource = "";
+        String resourceId = "";
 
         Transaction transaction = transactionService.getTransaction(resource);
         if (transaction == null) {
             LOG.error("transaction not found");
             return null;
         }
 
-        return new DocumentInfo();
+        String accountLink = Optional.of(transaction)
+                .map(Transaction::getResources)
+                .map(e -> e.get(resourceId))
```

Comment:
would change from e to resources, makes it more readable.

Reply:
good point, changed